### DISCUSSION TITLE
Cleaning schedule with more than one congregation

### DISCUSSION THREAD

**Discussion by fschein** [sws2apps/organized-app#3206] [Ideas]
Hi, is everyone well? I hope so! 

Regarding the cleaning assignments, I don't know how it will be done, nor how it is done around the world, but our cleaning works as follows:

- Cleaning at every meeting;
- Weekly cleaning;
- Quarterly cleaning.

We have two congregations that use the Kingdom Hall.

At each meeting, one service group cleans the hall that week.
For example: this week my group cleans the midweek meeting and the weekend meeting. Next week, the next group on the list cleans.

Each congregation does the weekly cleaning, alternating the service groups from each congregation, as well as the quarterly cleaning. It's important to be able to inform the app that there is one or more other congregation that participates in the weekly and quarterly cleaning in order to organize it better.

It would be interesting if we had a way of configuring and automating the scheduling, so that you only had to print out the next schedules.

**Comment by ux-git** [comment]
Hi @fschein, thanks for sharing your experience! 🙏 

Yes, such scenario adds some complexity for the scheduling process on both interface and code levels. I'll try to think how/if this can be implemented while being user friendly.
